clj-refactor.el

```diff
@@ -3365,6 +3366,17 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-create-fn-from-e
   (when (string-match "[^/]+$" s)
     (substring s (car (match-data)) (car (cdr (match-data))))))
 
+(defun cljr--dashed-words (s)
+  "Take the string S and replace all the word separators with '-'
+and make the whole string lower-cased."
+  (with-temp-buffer
+    (insert s)
+    (beginning-of-buffer)
+    (while (not (eobp))
+      (subword-forward)
+      (insert " "))
+    (mapconcat 'identity (split-string (downcase (buffer-string))) "-")))
```

Comment:
Looks OK to me.  If it works as it should I don't see a huge point in bikeshedding implementation details we can change at will :)